src/styles/base.scss

```diff
@@ -328,7 +334,33 @@ mat-chip {
 }
 
 
+
+/* Refresh list page */
+
+.update-page {
+  display: block;
+  background: $dark-gray;
+  width: 100%;
+  overflow: auto;
+  padding: 10px;
+  text-align: center;
+  color: #fff;
+  position: absolute;
+  width: auto;
```

Comment:
If this, then remove `width: 100%;` above.

Reply:
done